### PR TITLE
Basic guide first draft

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _basic/getting-started/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Since this article is about setting up a Codeship Basic project, you'll want to be sure to select the Basic infrastructure when adding your project.
+
+![Select Basic Infrastructure]({{ site.baseurl }}/images/basic-guide/select-infra.png)
+
+## Configuring Your Setup Commands
+
+After selecting the Basic infrastructure for your project, Codeship will next ask you to configure your setup and test commands.
+
+First, you will configure your setup commands. Setup commands are the commands you need the be able to run your tests and deployments. Typically these are things like fetching dependencies, seeding databases and installing packages.
```
Comment (mlocher) [inline _basic/getting-started/getting-started.md]: ... you need _to_ be able to ...

Comment (mlocher) [inline _basic/getting-started/getting-started.md]: We might want to be careful with _installing packages_.

File: _basic/getting-started/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Next, you will enter in your test commands. These are all tests you want to have run in your CI/CD pipeline, and all deployments you configure will be contingent on these tests passing.
+
+![Setup Commands on Codeship Basic]({{ site.baseurl }}/images/basic-guide/test-commands.png)
+
+## Configuring Your Deployments
+
+Now that you've defined your setup and test commands, you'll want to define your deployment pipelines. Deployment pipelines run only when a build's setup and tests commands have completed successful _and_ only when the branch defined for the deployment is matched. We call them deployment pipelines rather than deployments because you can have different deployment destinations - perhaps staging and master environments - triggered by different branches.
+
+Defining the branches that will trigger your deployment pipelines is the first step. You can either match a specific branch - i.e. `master` - or you can choose to match any branch that _starts with_ a specific string.
```
Comment (mlocher) [inline _basic/getting-started/getting-started.md]: We might want to link to https://documentation.codeship.com/basic/getting-started/wildcard-deployment-pipelines/ for the wildcard deployments.

File: _basic/getting-started/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+![Custom Script Deployment on Codeship Basic]({{ site.baseurl }}/images/basic-guide/custom-script-deployment.png)
+
+From there you will be presented with a command window, just like when you entered your setup and test commands, in which you can define what deployment scripts or commands you would like to run.
+
+![Custom Script Deployment on Codeship Basic]({{ site.baseurl }}/images/basic-guide/custom-script-deployment-bash.png)
+
+### Multiple Deployment Steps
+
+It's worth noting that for each deployment pipeline, you can add multiple deployments or multiple deployment steps. For instance, you can have one deployment on a `master` pipeline that runs your deployment scripts followed by another pipeline that runs notification scripts. To add multiple steps or deployments to a pipeline, just click on an additional deployment target and specify as needed.
```
Comment (mlocher) [inline _basic/getting-started/getting-started.md]: We might want to mention that reordering the individual deployments is possible via drag and drop on the numbers.

File: _basic/getting-started/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+![Custom Script Deployment on Codeship Basic]({{ site.baseurl }}/images/basic-guide/multiple-deployments.png)
+
+## Common Questions
+
+### Caching
+
+Codeship Basic has an automatic, built-in dependency cache, meaning we cache the packages directory for the most common frameworks, such as Node and Rails, to speed up future builds. You can clear your dependency cache at any time via the sidebar.
```
Comment (mlocher) [inline _basic/getting-started/getting-started.md]: Node_JS_, but it's not really a framework. Maybe mention dependency management systems like NPM and Rubygems

File: _basic/getting-started/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+![Reset Dependency Cache]({{ site.baseurl }}/images/basic-guide/reset-dependency-cache.png)
+
+### Skipping Builds
+
+By using the string `skip-ci` in your commit message, you can instruct Codeship not to run a build for a particular commit. You can learn more about this [here]({{ site.baseurl }}{% link _basic/getting-started/skipping-builds.md %})
```
Comment (mlocher) [inline _basic/getting-started/getting-started.md]: should be `--skip-ci` (the dashes are important as they are part of the regexp pattern)

File: _basic/getting-started/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+### Parallelizing
+
+Codeship offers the option to upgrade your Basic account with additional parallel test pipelines, allowing you to run multiple test commands simultaneously as a way to great speed up your build.
```
Comment (mlocher) [inline _basic/getting-started/getting-started.md]: great_ly_?

**Comment by ethangj** [comment]
@mlocher Updated!

**Pull request merged**